github/create_labels/create_labels.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+
+# This script creates a new label at Github which can be used for labeling
+# issues and pull requests.
+#
+# For creating the hook you need a GitHub token with appropriate permissions,
+# create your token at https://github.com/settings/tokens/new.
+# Pass the token via "GH_TOKEN" environment variable.
+
+# install missing gems
+if !File.exist?("./vendor")
+  puts "Installing needed Rubygems to ./vendor/bundle ..."
+  `bundle install --path vendor/bundle`
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require "octokit"
+
+github = Octokit::Client.new(:access_token => ENV["GH_TOKEN"])
```

Comment:
what about checking that GH_TOKEN is set? just usability, to write reasonable error msg to stderr